src/sentry/snuba/metrics/query_builder.py

```diff
@@ -152,17 +152,34 @@ def resolve_tags(
                 ],
             )
         elif input_.function == "match":
-            column = input_.parameters[0]
+            input_first_param = input_.parameters[0]
+
+            # For now, we only support as the first parameter of "match" only the direct column and the "ifNull"
```

Comment:
This comment doesn't follow the rule:  
> Focus on the why. The best code comments are the ones you don’t need.
So it just repeat the implementation.

IMO we should or remove it or add why do we want to support only 2 options and raise exception.

cc @RaduW wdyt?